modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -273,6 +273,32 @@ test.describe('Manage object definitions through Model Builder', () => {
 		await apiHelpers.objectAdmin.deleteObjectFolder(objectFolder.id);
 	});
 
+	test('navigate to edit object definition page', async ({
+		context,
+		modelBuilderPage,
+		page,
+	}) => {
+		const pagePromise = context.waitForEvent('page');
```

Comment:
You only need to start waiting for this event right before calling `openPageViewButton.click():`. Then we would have something like this:
```
const pagePromise = context.waitForEvent('page');

await modelBuilderPage.openPageViewButton.click();

const editObjectDefinitionPage = await pagePromise;
```